python/src/com/jetbrains/python/PyGotoSymbolContributor.java

```diff
@@ -45,6 +42,7 @@
     symbols.addAll(PyModuleNameIndex.getAllKeys(project));
     symbols.addAll(StubIndex.getInstance().getAllKeys(PyFunctionNameIndex.KEY, project));
     symbols.addAll(StubIndex.getInstance().getAllKeys(PyVariableNameIndex.KEY, project));
+    symbols.addAll(StubIndex.getInstance().getAllKeys(PyClassAttributesIndex.KEY, project));
```

Comment:
We don't suggest navigating to instance attributes here. I would either suggest both instance and class-level attributes or neither of these.